frontend/src/components/header/Header.jsx

```diff
@@ -1,44 +1,90 @@
-import React from 'react';
+import React, { useState, useEffect, useRef } from 'react';
 import { ReactComponent as Logo } from 'assets/images/logo.svg';
+import { ReactComponent as Menu } from 'assets/images/menu.svg';
```

Comment:
this line break build, seems like `menu.svg` doesn't exist at all, could you please double check and fix?

Reply:
alright

Reply:
its breaking insides assets/images so i have moved it to assets/icons